src/components/Map/index.tsx

```diff
@@ -169,26 +175,52 @@ const Map = () => {
     };
   }, [airQualityByCity]);
 
+  const handleClickMarker = useCallback((city: Element) => {
+    setCity(city.id);
+    if (city instanceof HTMLElement) {
+      city.dataset.finedustgrade
+        ? setFineDustScale(+city.dataset.finedustgrade)
+        : '';
+      city.dataset.ultrafinedustgrade
+        ? setUltraFineDustScale(+city.dataset.ultrafinedustgrade)
+        : '';
+    }
+    onOpen();
+  }, []);
+
+  const handleMouseOverMarker = useCallback((city: HTMLElement) => {
+    if (city.parentElement)
+      city.parentElement.style.zIndex = ZINDEX_MARKER_MOUSE_OVER;
+    city.style.color = COLOR_MARKER_MOUSE_OVER;
+  }, []);
```

Comment:
```suggestion
  const handleMouseOverMarker = useCallback((city: HTMLElement) => {
    if (city.parentElement) {
       city.parentElement.style.zIndex = ZINDEX_MARKER_MOUSE_OVER;
    }
    
    city.style.color = COLOR_MARKER_MOUSE_OVER;
  }, []);
```
가독성을 위해 중괄호를 작성하는 것이 좋을 것 같아요.